blog/2024-02-20-grpc-performance-improvements/index.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: gRPC-Driven Performance Improvements in Weaviate
+slug: grpc-performance-improvements
+authors: [dirk, leonie, jp]
+date: 2024-02-20
+image: ./img/hero.png
+tags: ['engineering', 'concepts']
+description: "Learn how to fine-tune Cohere's reranker and generate synthetic data using DSPy!"
```

Comment:
What about "Learn how gRPC improves import and query speeds in the v4 Weaviate client"

Reply:
Perfect